twake/frontend/src/app/services/login/LoginService.ts

```diff
@@ -0,0 +1,88 @@
+import Api from '../Api';
+import { TwakeService } from '../Decorators/TwakeService';
+import Globals from '../Globals';
+import { Login as DeprecatedLogin } from './login';
+import AuthService from '../Auth/AuthService';
+import { AuthProvider } from '../Auth/provider/AuthProvider';
+
+type ExternalProviderType = 'openid' | 'cas' | string;
```

Comment:
@RomaricMourgues do we still want to support CAS and OPENID? If not I will remove all the related code